app/lib/methods/handleMediaDownload.ts

```diff
@@ -73,10 +73,8 @@ const getExtension = (type: MediaTypes, mimeType?: string) => {
 	if (mimeType === 'audio/mpeg') {
 		return 'mp3';
 	}
-	// Audios sent by Android devices are in the audio/aac format, which cannot be converted to mp3 by iOS.
-	// However, both platforms support the m4a format, so they can maintain the same behavior.
 	if (mimeType === 'audio/aac') {
-		return 'm4a';
+		return 'aac';
```

Comment:
Can you make sure that this if is still relevant?
There's a chance `'react-native-mime-types'` already returns the correct extension